flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -40,4 +36,14 @@ private FlinkConfigOptions() {
           .intType()
           .defaultValue(100)
           .withDescription("Sets max infer parallelism for source operator.");
+
+  public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_INFER_SOURCE_LOCALITY =
+      ConfigOptions.key("table.exec.iceberg.infer-source-locality")
+          .booleanType()
+          .defaultValue(true)
```

Comment:
Is it possible to remove the default here? The purpose of this is to override the default (hdfs=on, everything else=off) and disable locality in splits, so I don't think that adding a default here makes sense.

Reply:
done